### ISSUE TITLE
about OsirisExtenderSettings.json

### ISSUE THREAD

**Issue by blackdarks** [Norbyte/ositools#83]
### Operating System

Windows

### Extender Version

v55

### Game Version

v3.6.117.3735

### Bug Summary

Why can't I find OsirisExtenderSettings.json file in defed / bin directory ?

**Comment by Wambosa** [comment]
I had to create this in `{my-steam-installation-root}/steam/steamapps/common/Divinity\ Original\ Sin\ 2/DefEd/bin/OsirisExtenderSettings.json`

the contents I chose were
```
{
  "EnableAchievements": true
}
```

Now I got the thing to not crash, but uncertain if it actually has enabled achievements

Also side note on steamdeck, mods needed to be installed to `~/.steam/steam/steamapps/compatdata/435150/pfx/drive_c/users/steamuser/Documents/Larian\ Studios/Divinity\ Original\ Sin\ 2\ Definitive\ Edition/Mods/`

**Comment by Canowyrms** [comment]
@Wambosa I agree that it would be nice to include `OsirisExtenderSettings.json` in the release, or at least be documented somewhere for us normies.

Were you able to confirm if achievements are working? I was testing with v57 this week and can confirm achievements worked for me (however, I am on Windows). I installed this tool, duplicated my save file, enabled gift bag features, got the warning achievements would be disabled, then got a couple of achievements as I progressed the story.
